### DISCUSSION TITLE
Unreasonable lhf and shf generated when using dephy format over ocean with prescribed Ts

### DISCUSSION THREAD

**Discussion by XiaSun-Atmos** [NCAR/ccpp-scm#336] [Cases, Initial Conditions, Forcing] [answered]
[](url)Hi,
I was using CCPP SCM over the ocean with forcing in DEPHY format. When using specified surface fluxes, the lhf and shf look reasonable. But when I tried to use prescribed skin temperature, i.e. I set sfc_flux_spec be false. It is generating unreasonable values for lhf and shf. 
![Screen Shot 2022-07-01 at 10 55 40 AM](https://user-images.githubusercontent.com/58949533/176938125-0e9ed315-f448-4be9-bf1c-b8013ad4b1c7.png)

Here are the namelist I used for prescribed Ts, not prescribed surface fluxes: 
```
$case_config
case_name = 'icon_lat100_lon99',
runtime = 10800,
input_type = 1,
mom_forcing_type=2
sfc_flux_spec = .false.,
sfc_type=0,
thermo_forcing_type=2,
reference_profile_choice = 2,
C_RES = 768,
$end
```

Any comment is appreciated. Thanks.
Xia

**Comment by grantfirl** [comment, marked as answer]
Hi Xia,

When using DEPHY-formatted cases, many case configuration namelist variables are ignored or otherwise incompatible. In the case configuration namelist you posted, `runtime`, `mom_forcing_type`, `sfc_flux_spec`, `sfc_type`, and `thermo_forcing_type` are all incompatible with the DEPHY case files. Changing the forcing, surface type, runtime, or whether you're using specified surface fluxes or prescribed SSTs should all be done by changing the DEPHY-formatted case data file. E.g., if you need to try both prescribed fluxes and prescribed SSTs for the same case, you'll need 2 different DEPHY-formatted case data files with the appropriate global attributes and data to run your desired configurations.
